symengine/series.h

```diff
@@ -187,7 +188,7 @@ class SeriesBase : public SeriesCoeffInterface
                                      unsigned int prec)
     {
         if (s == 0)
-            throw std::runtime_error("Series::series_invert: division by zero");
+            throw DivisionByZero();
```

Comment:
Can be done, but the problem is how to send that to the Ruby Wrapper. Shall I create a child class of the `DivisionByZero`?
Eg: `DivisionByZeroSeriesInvert` or something similar.